### DISCUSSION TITLE
At a Prisma level, a connection between two models is always represented by a relation field on each side of the relation.

### DISCUSSION THREAD

**Discussion by atyapin** [prisma/prisma#18673] [Q&A] [answered]
Hello!
I can see in documentation [Relations](https://www.prisma.io/docs/concepts/components/prisma-schema/relations) that

> At a Prisma level, a connection between two models is **always** represented by a relation field on **each side** of the relation.

I am wondering why it's required to maintain two sides as it seems we have enough information at "many" side.
And it becomes not maintainable at "one side" model when we have quantity of "many side" models growing as database scheme is growing.

**Comment by nurul3101** [comment, marked as answer]
Hello @atyapin 👋 

In Prisma, relations are bidirectional. This design choice was considered because it provides flexibility and ensures consistency in the data model. It makes it easier to work with related data, as you can access it from either side of the relation.

However, as you said, it can become complex as the database schema grows.

We have feature requests to simplify this:
- https://github.com/prisma/prisma/issues/2018

I would recommend adding a 👍 to the above feature request so that our product team can prioritise it.

There is also this related GitHub Issue that would help in providing additional context:
- https://github.com/prisma/prisma/issues/3045

Hope this helps, please feel free to let us know in case you have any additional questions.
